tasks/internal_reconcile_report_orphan/src/adapters/api/aws.py

```diff
@@ -0,0 +1,166 @@
+    """
+    Checks for the lambda environment variable.
+    Args:
+        env_name (str): The environment variable name set in lambda configuration.
+    Raises: KeyError in case the environment variable is not found.
+    """
+    try:
+        env_value = os.environ[env_name]
+        if len(env_value) == 0 or env_value is None:
+            raise KeyError(f"Empty value for {env_name}")
+    except KeyError:
+        LOGGER.error(f"{env_name} environment value not found.")
+        raise
+
+    return env_value
+
+
+def create_http_error_dict(
+        error_type: str, http_status_code: int, request_id: str, message: str
+) -> Dict[str, Any]:
+    """
+    Creates a standardized dictionary for error reporting.
+    Args:
+        error_type: The string representation of http_status_code.
+        http_status_code: The integer representation of the http error.
+        request_id: The incoming request's id.
+        message: The message to display to the user and to record for debugging.
+    Returns:
+        A dict with the following keys:
+            'errorType' (str)
+            'httpStatus' (int)
+            'requestId' (str)
+            'message' (str)
+    """
+    LOGGER.error(message)
+    return {
+        "errorType": error_type,
+        "httpStatus": http_status_code,
+        "requestId": request_id,
+        "message": message,
+    }
+
+
+OS_ENVIRON_DB_CONNECT_INFO_SECRET_ARN_KEY = "DB_CONNECT_INFO_SECRET_ARN"  # nosec
+
+
+def handler(
+        event: Dict[str, Union[str, int]], context: Any
+) -> Union[List[Dict[str, Any]], Dict[str, Any]]:
+    """
+    Entry point for the internal_reconcile_report_orphan Lambda.
+    Args:
+        event: See schemas/input.json for details.
+        context: An object provided by AWS Lambda. Used for context tracking.
+
+    Environment Vars:
+        DB_CONNECT_INFO_SECRET_ARN (string):
+            Secret ARN of the AWS secretsmanager secret for connecting to the database.
+            See shared_db.py's get_configuration for further details.
+
+    Returns:
+        See schemas/output.json
+        Or, if an error occurs, see create_http_error_dict
+            400 if input does not match schemas/input.json.
+            500 if an error occurs when querying the database.
+    """
+    try:
+        LOGGER.setMetadata(event, context)
+
+        try:
+            _VALIDATE_INPUT(event)
+        except JsonSchemaException as json_schema_exception:
+            return src.adapters.api.aws.create_http_error_dict(
+                "BadRequest",
+                HTTPStatus.BAD_REQUEST,
+                context.aws_request_id,
+                json_schema_exception.__str__(),
+            )
+
+        db_connect_info = orca_shared.database.adapters.api.aws.get_configuration(
+            check_env_variable(OS_ENVIRON_DB_CONNECT_INFO_SECRET_ARN_KEY),
+            LOGGER
+        )
+
+        storage_adapter = src.adapters.storage.rdbms.StorageAdapterPostgres(
```

Comment:
Since you import `StorageAdapterPostgres` wouldn't this be `storage_adapter = StorageAdapterPostgres(create_user_uri(db_connect_info, LOGGER))` or is there a reason for the full pathing?